composer.json

```diff
@@ -172,6 +172,7 @@
 		"ci:yaml:lint": "find . ! -path '*.Build/*' ! -path '*node_modules/*' \\( -name '*.yaml' -o -name '*.yml' \\) | xargs -r php ./.Build/bin/yaml-lint",
 		"coverage:create-directories": "mkdir -p .Build/logs .Build/coverage",
 		"docs:generate": "docker run --rm --pull always -v $(pwd):/project -it ghcr.io/typo3-documentation/render-guides:latest --config=Documentation",
+		"fix": "@fix:php",
```

Comment:
Please also add a description for this command. This is what I use in my other projects:

```
"fix": "Runs all automatic code style fixes.",
```

Reply:
Done.